src/plugins/condition/criterion/TelemetryCriterion.js

```diff
@@ -51,7 +51,11 @@ export default class TelemetryCriterion extends EventEmitter {
     }
 
     initialize() {
-        this.telemetryObjectIdAsString = this.openmct.objects.makeKeyString(this.telemetryDomainObjectDefinition.telemetry);
+        this.telemetryObjectIdAsString = "";
+        if (this.telemetryDomainObjectDefinition?.telemetry !== "" && this.telemetryDomainObjectDefinition?.telemetry !== null) {
+            this.telemetryObjectIdAsString = this.openmct.objects.makeKeyString(this.telemetryDomainObjectDefinition.telemetry);
```

Comment:
Right now the default is that the `this.telemetryDomainObjectDefinition.telemetry` is `""`, so it seems more efficient to not run the `makeKeyString` code if that's the case instead of running it, having it error out, and then setting the return value to an empty string.

Reply:
Agreed, I think checking for null before passing the telemetry is better than having the api error out

Reply:
OK.
Also, I believe optional chaining will always return undefined if the reference is `nullish` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining
So we should check for `undefined`.